app/controllers/users/sessions_controller.rb

```diff
@@ -4,6 +4,7 @@ class Users::SessionsController < Devise::SessionsController
   before_action :admin_logout_required
   skip_before_action :store_location
 
+  # POST /users/login 
```

Comment:
You don't have to add these comments, btw. They're just holdovers from early auto-generated Rails scaffolding, and we don't use them consistently.

Reply:
I know, I just thought it was nice to have as documentation for the next time we'd have to make changes to this controller.